### PR TITLE
feat: support clv wallet

### LINKED ISSUE #4317: Add suport for CLV Extension Wallet
Add support for new wallet, seems they have 60k+ installation by chrome store 

- https://docs.clv.org/use-clv-wallet/clv-extension-wallet
- https://chrome.google.com/webstore/detail/clv-wallet/nhnkbkgjikgcigadomkphalanndcapjk

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #4317
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Comment by yangwao** [comment]
ooo

File: utils/config/wallets/BaseDotsamaWallet.ts [hunk 1]
```diff
@@ -107,7 +107,9 @@ export class BaseDotsamaWallet implements Wallet {
       callback(undefined)
       return null
     }
-
+    if (!this._extension.accounts.subscribe) {
+      return null
+    }
```
Comment (roiLeo) [inline utils/config/wallets/BaseDotsamaWallet.ts:112]: What's this? does it fix something?

**Review by roiLeo — COMMENTED** [review]
Ooooh! clv is clover wallet, I see that it was partially planned to integrate it.

✅ code lgtm
🟥 didn't test it yet

File: utils/config/wallets/BaseDotsamaWallet.ts (hunk 1 above)
Comment (Jarsen136) [reply utils/config/wallets/BaseDotsamaWallet.ts:112]: yes, the clv extension wallet should have implemented the subscribe function. : )

**Review by prachi00 — APPROVED** [review]

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 25.34 USD/KSM ~ 1.973 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0x5c2b894c4c2655216eab81533a462bb0bab0671cafe4fd61eff63727f376a75b](https://kusama.subscan.io/extrinsic/0x5c2b894c4c2655216eab81533a462bb0bab0671cafe4fd61eff63727f376a75b)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**